spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/ui/UiController.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.data.ui;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+/**
+ * A simple controller to trigger a redirect to the index page of the admin ui (which comes packaged as a dependency).
+ *
+ * @author Eric Bottard
+ */
+// Note how this controller is NOT a RestController
```

Comment:
should include an explanation of *why* (and does this belong in actual JavaDoc?)